### ISSUE TITLE
Drop Jython support

### ISSUE THREAD

**Issue by aaltat** [robotframework/SeleniumLibrary#1451]
It is not possible to test Jython in Travis anymore. Therefore support for Jython is dropped in SeleniumLibrary 4.1

**Comment by pekkaklarck** [comment]
This is a pretty strange decision in my opinion. On code level there's practically no extra work supporting Jython as long as Python 2 is supported, and testing Jython locally at the pre-release phase isn't a big task. Updating docs regarding to Jython supports is probably more work.

**Comment by aaltat** [comment]
I also have the same Ubuntu LTS version as Travis has. It's possible to install Jython and running Python code from the shell works. But something in Java/Jython/Selenium combination or something else prevents Selenium to start browser properly and testing with Jython is not possible. I already spend few hours trying to figure out the problem but without any luck. The only solution that I see to continue supporting Jython is to spin up VM with Ubuntu 14.04, install all the required things and run tests there. It's about half a day work and that is time I don't want to invest for Jython.

Python 2 will be supported until end of year and it's not very likely that there are changes that would break Jython. But because I have lost visibility for Jython I don't want to support it.

**Comment by pekkaklarck** [comment]
Sounds strange. It's hard to setup environment on CI that you don't directly control, but if setting environment locally doesn't work either I agree spending too much time with it isn't worth the effort. You initially mentioned only CI and dropping support of some platform due to CI issues sounds odd.

Perhaps you could ask the community is someone interested to test SL with Jython now and then and especially before releases. I got a feeling it still has users and someone having an environment where it works could easily run tests.

**Comment by aaltat** [comment]
Asking help is a good idea, did send message in slack and will post it later to other channels too. 

But yes, wording could have been better. For a long time, I have trusted Travis to run test in Jython. Only when I did try to upgrade the VM in travis, I did notice that Jython had stopped working also locally. SeleniumLibrary project does not run in to Jython specific problems that often, I would assume that Robot Framework core finds Jython specific problem more often.
